arrays/arrays.c

```diff
@@ -60,13 +60,23 @@ void destroy_array(Array *arr) {
 void resize_array(Array *arr) {
 
   // Create a new element storage with double capacity
+  char **newElements = calloc((2 * arr->capacity), sizeof(char *));
 
   // Copy elements into the new storage
+  for(int i = 0; i < arr->count; i++){
+    newElements[i] = arr->elements[i];
+  }
 
   // Free the old elements array (but NOT the strings they point to)
```

Comment:
This for loop appears to be unnecessary as it frees the strings the old elements points to.